ui/desktop/app/routes/scopes/scope/projects/sessions/session.js

```diff
@@ -17,25 +17,11 @@ export default class ScopesScopeProjectsSessionsSessionRoute extends Route {
    * Load a session
    * @param {object} params
    * @param {string} params.session_id
-   * @return {{session: SessionModel, target: TargetModel}}
+   * @return {SessionModel}
    */
-  async model({ session_id }) {
-    let target = null;
-    const session = await this.store.findRecord('session', session_id, {
+  model({ session_id }) {
+    return this.store.peekRecord('session', session_id, {
```

Comment:
Was this meant to be a `peekRecord` now? It doesn't support `reload: true` but I'm also curious as to why the switch

Reply:
Good call out on the `reload: true`. I updated this to `peedRecord` because we will never be able to access the session until after it has been created and loaded into the data store. So we don't need to use `findRecord`. I even tested with closing the desktop client with active sessions and opening the client back up. However, when you close the desktop client, it shuts down all active or pending sessions.

Reply:
I don't think that's always true as you pointed out with the CLI. But also especially as we move to paginated list responses where not all resources are guaranteed to be loaded into the datastore from the route hierarchy so we can't assume they're already loaded.

I would save the trouble and just always load singular resources

Reply:
Fixed.